jobbergate-agent/jobbergate_agent/jobbergate/schemas.py

```diff
@@ -78,3 +78,45 @@ class SlurmJobData(pydantic.BaseModel, extra="ignore"):
     job_state: Optional[str] = None
     job_info: Optional[str] = None
     state_reason: Optional[str] = None
+
+
+class InfluxDBMeasurement(TypedDict):
```

Comment:
It makes it really hard for someone reading the code to understand what they are and how they are different. Also, if you see them in other contexts it can be confusing about which is which.

Reply:
check [3f0b6f1](https://github.com/omnivector-solutions/jobbergate/pull/663/commits/3f0b6f13f92ec33eb4fb3d54fdc4dd39ee5a642f)